src/FileSystemUtilities.js

```diff
@@ -63,6 +64,9 @@ export default class FileSystemUtilities {
       }
       type = "file";
     }
+    if (pathIsAbsolute(src) && process.platform !== "win32") {
```

Comment:
Reading through [the docs](https://nodejs.org/api/fs.html#fs_fs_symlink_target_path_type_callback) again it actually looks like a relative path would be fine on Windows, since it will be automatically converted to absolute?

> Note that Windows junction points require the destination path to be absolute. When using 'junction', the target argument will automatically be normalized to absolute path.

And if the path coming in here is always absolute can we just skip this whole conditional?

Then we could also avoid adding a dep on `pathIsAbsolute`.